### PR TITLE
TRACK-82 Uploading SQL dumps to S3

### PR DESCRIPTION
**Pull request by tomca32**

### CODE REVIEW

**Comment by tomca32** [comment]
@j16r this make sense to you?

File: deploy/roles/replica/tasks/main.yml [hunk 1]
```diff
@@ -25,6 +25,31 @@
     dest: /mnt/data/db/recovery.conf
   notify: restart postgresql
 
+- name: Install Pip
+  apt:
+    name: python-pip
```
Comment (j16r) [inline deploy/roles/replica/tasks/main.yml]: What do you need pip for?

Comment (tomca32) [inline deploy/roles/replica/tasks/main.yml]: For AWS CLI. It's only distributed over pip

Comment (j16r) [inline deploy/roles/replica/tasks/main.yml]: Ahh 👍

**Pull request merged**